.github/workflows/nightly-pekko-1.1-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Nightly Testing with Pekko 1.1
+
+permissions: {}
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Build and Test
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { java-version: 8,  scala-version: 2.12, sbt-opts: '' }
+          - { java-version: 8,  scala-version: 2.13, sbt-opts: '' }
+          - { java-version: 8,  scala-version: 3.3,  sbt-opts: '' }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
+
+      - name: Setup JDK ${{ matrix.java-version }}
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: ${{ matrix.java-version }}
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6
+
+      - name: Run tests with Scala ${{ matrix.scala-version }} and Java ${{ matrix.java-version }}
+        run: sbt -Dpekko.build.pekko.version=main "++${{ matrix.scala-version }} test" ${{ matrix.sbt-opts }}
```

Comment:
with sbt package ？

Reply:
I need to run the test

Reply:
Package will run the test too,IIRC, what about test;package?

Reply:
why do I need to package? it is the tests that we need to run

Reply:
It can test the doc format